lib/ansible/modules/monitoring/sensu_silence.py

```diff
@@ -0,0 +1,387 @@
+#!/usr/bin/python
+'''
+-*- coding: utf-8 -*-
+
+(c) 2017, Steven Bambling <smbambling@gmail.com>
+
+This file is part of Ansible
+
+Ansible is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+Ansible is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+'''
+
+try:
+    import json
+except ImportError:
+    import simplejson as json
+
+# import module snippets
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.urls import fetch_url
+
+ANSIBLE_METADATA = {'status': ['preview'],
+                    'supported_by': 'community',
+                    'version': '0.1'}
+
+DOCUMENTATION = '''
+---
+module: sensu_silence
+version_added: "2.2"
+author: Steven Bambling(@smbambling)
+short_description: Manage Sensu slience entries
+description:
+  - Create and clear (delete) a silence entries via the Sensu API
+    for subscriptions and checks.
+options:
+  check:
+    description:
+      - Specifies the check which the silence entry applies to.
+    required: false
```

Comment:
Please remove `required: false` and `default: []` lines as they are the defaults.

The following page explains this https://docs.ansible.com/ansible/dev_guide/developing_modules_documenting.html